src/components/pages/career-details/career-details.tsx

```diff
@@ -42,13 +35,6 @@ export const CareerDetails = ({
     ref?.current?.scrollIntoView({ behavior: 'smooth', block: 'start' });
   };
 
-  const location = useLocation();
-  const searchParams = new URLSearchParams(location.search);
-  const personioChannel = searchParams.get('_pc');
-  const utmChannel = UTM_CHANNEL_MAP[searchParams.get('utm_source') ?? ''];
-
-  const recruitingChannel = utmChannel ?? personioChannel ?? DEFAULT_CHANNEL_ID;
-
```

Comment:
Do we still need information about the recruitingChannel or is this related to personio?

Reply:
Yes that would be could. We never implemented that on the website though. So remove it here and then let's come back. I would like to add a parameter to our job postings so we can share it on different channels and then know through which channel someone came. Could be website, email etc or also platform or person related. Let's discuss this on a separate PR though